cylc/flow/workflow_files.py

```diff
@@ -751,6 +751,16 @@ def _clean_check(opts: 'Values', reg: str, run_dir: Path) -> None:
         raise ServiceFileError(
             f"Cannot clean running workflow {reg}.\n\n{exc}"
         )
+    except CylcError:
+        db_path = (
+            run_dir / WorkflowFiles.Service.DIRNAME / WorkflowFiles.Service.DB
+        )
```

Comment:
I think this is catching the SSH error, I think we should really prevent it from happening.

```python
def _clean_check(opts: 'Values', reg: str, run_dir: Path) -> None:
    validate_workflow_name(reg)
    reg = os.path.normpath(reg)
    if get_contact_path(reg).exists() and not get_db_path(reg).exists():
       ...
```